backend/pom.xml

```diff
@@ -18,7 +18,8 @@
     <properties>
         <start-class>ai.verta.modeldb.App</start-class>
         <sonar.exclusions>src/main/java/ai/verta/modeldb/versioning/autogenerated/**/*</sonar.exclusions>
-        <otel.core.version>1.14.0</otel.core.version>
+        <otel.core.version>1.13.0</otel.core.version>
+        <otel.instrumentation.version>1.13.1-alpha</otel.instrumentation.version>
```

Comment:
Out of curiosity, what is the reason for the downgrade?

Reply:
at the time of writing, the instrumentation hadn't been updated to match 1.13 of the core. Looks like they've released in the meantime. I will update to the latest!

Reply:
done!